package.json

```diff
@@ -30,31 +30,32 @@
   "license": "Apache-2.0",
   "homepage": "https://github.com/rjsf-team/react-jsonschema-form",
   "devDependencies": {
-    "@babel/eslint-parser": "^7.22.15",
+    "@babel/eslint-parser": "^7.23.3",
     "@nrwl/nx-cloud": "^15.3.5",
-    "@types/estree": "^1.0.1",
-    "@types/jest": "^29.5.5",
-    "@types/node": "^18.17.14",
+    "@types/estree": "^1.0.5",
+    "@types/jest": "^29.5.11",
+    "@types/node": "^18.19.4",
     "@types/prettier": "^2.7.3",
-    "@types/react": "^17.0.65",
-    "@types/react-dom": "^17.0.20",
+    "@types/react": "^17.0.74",
+    "@types/react-dom": "^17.0.25",
     "@typescript-eslint/eslint-plugin": "^5.62.0",
     "@typescript-eslint/parser": "^5.62.0",
     "cross-env": "^7.0.3",
-    "eslint": "^8.48.0",
+    "eslint": "^8.56.0",
     "eslint-config-prettier": "^8.10.0",
-    "eslint-plugin-import": "^2.28.1",
-    "eslint-plugin-jsx-a11y": "^6.7.1",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "husky": "^8.0.3",
-    "jest-environment-jsdom": "^29.6.4",
+    "jest-environment-jsdom": "^29.7.0",
+    "jest-environment-jsdom-global": "^4.0.0",
```

Comment:
@heath-freenome why was this new dependency added?

Reply:
It was needed at some point to fix an issue with tests. And removing it now seems to not cause any problems.